src/applets/status/meson.build

```diff
@@ -38,6 +39,19 @@ applet_status_deps = [
     gvc.get_variable('libgvc_dep'),
     meson.get_compiler('c').find_library('m', required: false),
 ]
+else
+applet_status_deps = [
```

Comment:
It would probably be better to assign the array without the gnome-bluetooth dep, then add it to the array if the host machine isn't openbsd, rather than having two separate assignments.